CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 ### Scan
 
 - (imprv) [\#1900](https://github.com/bandprotocol/bandchain/pull/1900) Shorten marketcap amount on landing page
+- (feat) [\#1888](https://github.com/bandprotocol/bandchain/pull/1888) Binding Obi.js
```

Comment:
Please elaborate. For example "Added OBI bindings and patched scan to use OBI standard"